charts/tfy-karpenter-config/templates/karpenter-inferentia-provisioner.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.karpenter.inferentiaProvisionerSpec.enabled }}
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: inferentia-provisioner
+  namespace: karpenter
+spec:
+  weight: 10
+  {{- if .Values.karpenter.inferentiaProvisionerSpec.consolidation.enabled }}
+  consolidation:
+    enabled: true
+  {{- else }}
+  ttlSecondsAfterEmpty: {{ .Values.karpenter.inferentiaProvisionerSpec.ttlSecondsAfterEmpty }}
+  {{- end }}
```

Comment:
Not directly relevant to this PR, 
I have seen karpenter complains that in case of gpu provisioner it got both `consolidation` and `ttlSecondsAfterEmpty` keys and spec is invalid
Now considering this if else it should be impossible
so my theory is argocd has some weird bug where this if condition is not evaluated correctly

Reply:
its not actually or I am not pretty sure, but from argo side it first tries to check the new change and then remove the old one